src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -252,6 +247,16 @@ public static String getAttributeTableName(String clsName,
 		return eventsNotUsedForInference;
 	}
 
+	/**
+	 * Checks if incoming Event is manually inferred by checking inferredFrom referral.
+	 * @param event GKInstance -- Event instance being checked for inferredFrom referral.
+	 * @return boolean -- true if inferredFrom referral exists, false if not.
+	 * @throws Exception -- Thrown by MySQLAdaptor.
+	 */
+	private static boolean manuallyInferred(GKInstance event) throws Exception {
+		return event.getReferers(ReactomeJavaConstants.inferredFrom) != null;
+	}
+
```

Comment:
Maybe you could add a note that this check is only valid for instances coming from a curation database (i.e. gk_central or a slice) rather than a release database that uses `inferredFrom` for electronic inference.